ramp-database/ramp_database/tools/leaderboard.py

```diff
@@ -427,12 +439,19 @@ def get_leaderboard(
             columns = [
                 "team",
                 "submission",
+                "submission_id",
```

Comment:
Maybe let's just call it `id` so it would be a less wide column in the resulting html table? It's shown under a section of New submission, so there is no need to repeat "submission" in column names I think.